flexmeasures/ui/views/charts.py

```diff
@@ -43,13 +42,15 @@ class ChartRequestSchema(Schema):
     end_time = fields.DateTime(required=True)
     resolution = DurationField(required=True)
     show_consumption_as_positive = fields.Bool(missing=True)
-    forecast_horizon = DurationField(missing="PT6H")  # TODO: HorizonField
+    show_individual_traces_for = fields.Str(
+        missing="none", validate=lambda x: x in ("none", "schedules", "power")
+    )
+    forecast_horizon = DurationField(missing="PT6H")  # TODO: HorizonField?
```

Comment:
This todo can go. I see no need for a separate HorizonField. Both duration and horizon use the same ISO formatting.